README.md

```diff
@@ -0,0 +1,57 @@
+# python-cli-template
+
+A template repository for creating Python CLI applications.
+
+## App setup (delete this section and above after initial application setup)
+
+1. Rename "app" to the desired app name across the repo. (May be helpful to do a project-wide find-and-replace).
+2. Update Python version if needed.
+3. Install all dependencies to create initial Pipfile.lock with current versions.
```

Comment:
`Install all dependencies **with make install** to create initial Pipfile.lock with current versions.`?